google/cloud/storage/examples/storage_object_samples.cc

```diff
@@ -341,6 +341,71 @@ void ReadEncryptedObject(google::cloud::storage::Client client, int& argc,
   //! [read encrypted object] [END storage_download_encrypted_file]
   (std::move(client), bucket_name, object_name, base64_aes256_key);
 }
+
+void ComposeObject(google::cloud::storage::Client client, int& argc,
+                         char* argv[]) {
+  if (argc < 5) {
```

Comment:
nit: is there any reason we cannot support composing a single object?  If we can, why not `argc < 4`?

Reply:
Good call, will fix that!